core/stdlib/std.ncl

```diff
@@ -2148,6 +2148,28 @@
       "%%
       = fun field r => r."%{field}",
 
+    getor
+      : forall a. String -> a -> { _ : a } -> a
+      | doc m%%"
+        Returns the field of a record with given name or default value,
+        if there is no such field.
+
+        # Examples
+
+        ```nickel
+        std.record.get "tree" 3 { one = 1, two = 2 } =>
+          3
+
+        std.record.get "one" 11 { one = 1, two = 2 } =>
+          1
+        ```
+      "%%
+      = fun field default_value r =>
+        if std.record.has_field field r then
```

Comment:
```suggestion
        if %has_field% field r then
```

It's a little bit uglier, but in the stdlib we tend to use primop directly, to avoid a bit of indirection. Manual inlining if you wish.